row/src/Row.styles.css

```diff
@@ -0,0 +1,9 @@
+dockit-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+dockit-row *:not(:first-child) {
```

Comment:
I would avoid this because seems too specific to me

Reply:
do you have an idea on how to create a gap between components differently?

Reply:
FYI there is a gap value for flex https://caniuse.com/?search=flex%20gap

Reply:
`gap` sounds like a good solution here, I didn't know about it. Thanks!